frontend/src/metabase/entities/users/forms.js

```diff
@@ -90,18 +90,18 @@ export default {
     ],
   },
   password: {
-    fields: [
+    fields: () => [
```

Comment:
Maybe we could define our own tagged template literal that bundles this delayed evaluation?

```js
function delayedT(...args) {
  return () => t(...args)
}

delaytedT`something`
```

This would break if we tried to operate on the values as strings, but I think it would work if it all just ends up in JSX? There might also be negative performance implications.

Reply:
Interesting, yeah maybe. We might even be able to return an object (subclass of String?) with a `toString` method so if it gets coerced to a string it still works. The main difference is `typeof` would return `object`.